### DISCUSSION TITLE
sandpack-vue

### DISCUSSION THREAD

**Discussion by uyarn** [codesandbox/sandpack#67] [Ideas]
Hi @alexnm , is there any plan to provide a sanpack component of vue version? I would like to provide one.

**Comment by alexnm** [comment]
hey @uyarn. this is a long term plan, but we don't have it on our priority list at the moment since we haven't even finished the react version with all the things we wanted to do. But contributions are welcome of course. It just might be a separate project you can try building first on top of sandpack-client and we can talk later about integration. for now, we only officially have sandpack-react as a library

**Reply by uyarn** [reply]
Understood, I would create a personal repository. By the way, actually we are using the react version now and although the FileExplorer wasn't exported from the top( I think it wasn't completed) , we use it by a deep path, would you mind my doing some contribution in the FileExplorer component? such as add some icon or something like that to differentiate the Directory and File ~ or do you have any plan to convert it recently?

**Reply by alexnm** [reply]
no plans on that component, since it's not part of the standard sandpack layout. but feel free to suggest an improved design and I would also add it to the main export. if you reuse  the colors from the theme then it's also themable by default

**Comment by Destiner** [comment]
While waiting for the official support, you might want to check out [sandpack-vue](https://github.com/Destiner/sandpack-vue/). While it's not a 1:1 port of React components, I tried to add most of the features, like text editor integration and theming.

**Comment by jerrywu001** [comment]
Here is the vue3 version, which I modified from `sanpack-react` (without the `react-devtools` component)
link: [sandpack-vue3](https://www.npmjs.com/package/sandpack-vue3)

GitHub: https://github.com/jerrywu001/sandpack-vue3
